src/gc.c

```diff
@@ -189,6 +189,8 @@ NOINLINE uintptr_t gc_get_stack_ptr(void)
 #ifdef JULIA_ENABLE_THREADING
 static void jl_gc_wait_for_the_world(void)
 {
+    if (jl_n_threads > 1)
+        jl_wake_libuv();
```

Comment:
@yuyichao shouldn't libuv be in a gc-safe region?

Reply:
If we can spend time there without hitting other Julia runtime then yes. In which case we also need to make sure the transition back is emitted for all the callbacks.